README.md

```diff
@@ -98,3 +98,84 @@ There are two options when using the Avalanche-CLI:
 
 1. Use an official Subnet-EVM release: https://docs.avax.network/subnets/build-first-subnet
 2. Build and deploy a locally built (and optionally modified) version of Subnet-EVM: https://docs.avax.network/subnets/create-custom-subnet
+
+## Run in Docker
+
+The `subnet-evm` Docker image comes with AvalancheGo pre-installed, making it easy to run a node. You can find the latest image tags on [Docker Hub](https://hub.docker.com/r/avaplatform/subnet-evm/tags).
+
+### Configuration
+
+You can configure the `subnet-evm` Docker container using both environment variables and standard AvalancheGo config files.
+
+- **Environment Variables**: Use uppercase variables prefixed with `AVAGO_`. For example, `AVAGO_NETWORK_ID` corresponds to the `--network-id` flag in AvalancheGo.
+- **Config Files**: Configure as you would with the regular AvalancheGo binary using config files. Ensure `HOME` is set to `/home/avalanche` and mount the config directory with `-v ~/.avalanchego:/home/avalanche/.avalanchego`.
+
+### Data Persistence
+
+To persist data across container restarts, you need to mount the `.avalanchego` directory.
+
+- **Create Directory**: Run `mkdir -p ~/.avalanchego/staking` beforehand to create the necessary directories and avoid file system access issues.
+- **Set Permissions**: Run the container with `--user $(id -u):$(id -g)` to ensure proper access rights.
+- **Set Home and Mount**: Set `HOME` to `/home/avalanche` and mount the data directory with `-v ~/.avalanchego:/home/avalanche/.avalanchego`.
```

Comment:
This can be dangerous if people accidently set this from their host machine. We should either make it ultra clear that this is to be run from the container, or really don't mention this at all if this is not a blocker to run the container. If there is something wrong with the docker's own container, we should fix it in the dockerfile.

Reply:
Or perhaps have it as `/yourpath:/home/avalanche/.avalanchego` to highlight the host path is of your choosing.